app/controllers/items_controller.rb

```diff
@@ -1,8 +1,10 @@
 class ItemsController < ApplicationController
   def index
+    @items = Item.all
```

Comment:
On line 4, add:
authorize @item

Then, go to app/policies/item_policy.rb and add the following method:
def index?
  true
end